src/Mesh/babylon.groundMesh.ts

```diff
@@ -20,6 +22,14 @@
             return this._subdivisions;

```

Comment:
If subdivionX and Y are defined, this will return the wrong result (Same as my last comment)